pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest.java

```diff
@@ -1646,7 +1646,7 @@ public void persistentTopicsCursorResetAfterReset(String topicName) throws Excep
         });
 
         for (int i = 0; i < 10; i++) {
-            Message<byte[]> message = consumer.receive();
+            Message<byte[]> message = consumer.receive(5, TimeUnit.SECONDS);
```

Comment:
I am not sure it is a good idea to change to 5 seconds. Because it can cause flakiness if JVM pauses for 5 seconds.

Reply:
@sijie I'll double-check on this specific test, but on a lot of the tests, what was happening is that either the receive() call was hanging or timing out too soon (before all the messages were received.) The timeouts that were too short involved `consumer.receive(1, TimeUnit.SECONDS);` (if I remember correctly), and for one of them, `consumer.receive(3, TimeUnit.SECONDS);` was still too short. So, I increased it to `consumer.receive(5, TimeUnit.SECONDS);` which seemed to get the tests to pass consistently.